drylab/rpi-manual.md

```diff
@@ -8,98 +8,62 @@
-        - depending on what you need to use the box for, either works, one is just more secure and one is more flexible
-    - if using the black base, secure it to the white base with tape
-- place object on platform
-- place camera with desired lens into the adaptor, with lens facing down
-- preview camera view by running `preview.py` (refer to [Taking Photos/Videos] (**link**)
-    - note for orienting the view: the side where the ribbon comes out of the camera board is the bottom
-- adjust things as needed
-    - put platform on diff shelf
-    - turn on leds
-        - can adjust brightness of bright white leds
-        - if the object is reflective it may reflect the shadows in the spaces between the spiral of bright white leds. in that case using side lights may be better
-    - adjust focus and aperture
-        - to adjust: unscrew/loosen knob
-        - to fix: screw/tighten knob
-        - knob nearest to board (labelled with "NEAR" and "FAR") adjusts focus
-        - knob nearest to lens (labelled with "OPEN" and "CLOSE") adjusts aperture
-            - first find proper lighting with leds
-            - if and only if necessary, adjust aperture
-            - fix focus knob first to make adjusting aperture easier
-- secure camera settings
-    - fix/tighten knobs
-    - gently tape camera into place
-- close door
-- take photo(s)/video(s)!
-
-<!--
+1. Clean interior of box and platform thoroughly with ethanol
+    - *The door has a tendency to swing closed. If you need to keep the door open, tape it to a surface or prop it behind another object.*
+2. Place platform on desired shelf
+    - Make sure matte side of white base faces up
+    - *The white base is almost but not quite square so one side fits more snugly than the other. Depending on what you need to use the box for, either works; one orientation is just more secure and one is more flexible.*
+    - If using the black base, secure it to the white base with tape
+3. Place object on platform
+4. Place camera with desired lens into the adaptor, with lens facing down
+5. Preview camera view by running [`preview.py`](https://github.com/Klepac-Ceraj-Lab/RPiCamera/tree/main/scripts#preview)
+    - *For orienting the view - the side where the ribbon comes out of the camera board is the bottom of the photo*
+6. Adjust things as needed
+    - Put platform on different shelf
+    - Turn on leds
+        - *If the object is reflective it may reflect the spiral of bright white leds. in that case using side lights may be better*
+    - Adjust focus and aperture on the camera lens
+        - **Only adjust one aspect at a time. Leave the other knob fixed. This makes adjusting easier and faster.**
+        - To adjust: unscrew/loosen knob
+        - To fix: screw/tighten knob
+        - Knob nearest to board (labelled with "NEAR" and "FAR") adjusts focus
+        - Knob nearest to lens (labelled with "OPEN" and "CLOSE") adjusts aperture
```

Comment:
Currently the aperture is set on the largest diameter.

Reply:
I think we might want to do the opposite, unless light is an issue. With really wide apertures, it's much harder to find the right focus plane. And since we can increase exposure time without worrying about motion blur, I think that's a better way to get more light in the exposure

Reply:
Oh okay. That can be done. I'll play around with it when I get back to lab